clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/DefaultInputDataInfoProvider.java

```diff
@@ -118,7 +112,20 @@ public InputDataInfo validateInputAndGetInfo(String inputUri) throws Exception {
       pushJobSetting.inputDataSchemaString = pushJobSetting.inputDataSchema.toString();
       pushJobSetting.keySchema = extractAvroSubSchema(pushJobSetting.inputDataSchema, pushJobSetting.keyField);
     } else {
+      // try reading the file via sequence file reader. It indicates Vson input if it is succeeded.
```

Comment:
This comment is confusing. Earlier in the function, there’s an explicit check for a sequence file, but here it suggests the file might not be a sequence file. Why would that be the case?